### PR TITLE
Audit ES Translation of html5-and-css.json

### LINKED ISSUE #6501: Translation to Spanish: Verify and update translation of html5-and-css.json
- [x]  Say Hello to HTML Elements
- [x]  Headline with the h2 Element
- [x]  Inform with the Paragraph Element
- [x]  Uncomment HTML
- [x]  Comment out HTML
- [x]  Fill in the Blank with Placeholder Text
- [x]  Delete HTML Elements
- [x]  Change the Color of Text
- [x]  Use CSS Selectors to Style Elements
- [x]  Use a CSS Class to Style an Element
- [x]  Style Multiple Elements with a CSS Class
- [x]  Change the Font Size of an Element
- [x]  Set the Font Family of an Element
- [x]  Import a Google Font
- [x]  Specify How Fonts Should Degrade
- [x]  Add Images to your Website
- [x]  Size your Images
- [x]  Add Borders Around your Elements
- [x]  Add Rounded Corners with a Border Radius
- [x]  Make Circular Images with a Border Radius
- [x]  Link to External Pages with Anchor Elements
- [x]  Nest an Anchor Element within a Paragraph
- [x]  Make Dead Links using the Hash Symbol
- [x]  Turn an Image into a Link
- [x]  Add Alt Text to an Image for Accessibility
- [x]  Create a Bulleted Unordered List
- [x]  Create an Ordered List
- [x]  Create a Text Field
- [x]  Add Placeholder Text to a Text Field
- [x]  Create a Form Element
- [x]  Add a Submit Button to a Form
- [x]  Use HTML5 to Require a Field
- [x]  Create a Set of Radio Buttons
- [x]  Create a Set of Checkboxes
- [x]  Check Radio Buttons and Checkboxes by Default
- [x]  Nest Many Elements within a Single Div Element
- [x]  Give a Background Color to a Div Element
- [x]  Set the ID of an Element
- [x]  Use an ID Attribute to Style an Element
- [x]  Adjusting the Padding of an Element
- [x]  Adjust the Margin of an Element
- [x]  Add a Negative Margin to an Element
- [x]  Add Different Padding to Each Side of an Element
- [x]  Add Different Margins to Each Side of an Element
- [x]  Use Clockwise Notation to Specify the Padding of an Element
- [x]  Use Clockwise Notation to Specify the Margin of an Element
- [x]  Style the HTML Body Element
- [x]  Inherit Styles from the Body Element
- [x]  Prioritize One Style Over Another
- [x]  Override Styles in Subsequent CSS
- [x]  Override Class Declarations by Styling ID Attributes
- [x]  Override Class Declarations with Inline Styles
- [x]  Override All Other Styles by using Important
- [x]  Use Hex Code for Specific Colors
- [x]  Use Hex Code to Color Elements White
- [x]  Use Hex Code to Color Elements Red
- [x]  Use Hex Code to Color Elements Green
- [x]  Use Hex Code to Color Elements Blue
- [x]  Use Hex Code to Mix Colors
- [x]  Use Hex Code to Color Elements Gray
- [x]  Use Hex Code for Specific Shades of Gray
- [x]  Use Abbreviated Hex Code
- [x]  Use RGB values to Color Elements
- [x]  Use RGB to Color Elements White
- [x]  Use RGB to Color Elements Red
- [x]  Use RGB to Color Elements Green
- [x]  Use RGB to Color Elements Blue
- [x]  Use RGB to Mix Colors

### PR DESCRIPTION
**Pull request by justinian336**
Thanks to @vtamara for the corrections!
Closes #6501

### CODE REVIEW

File: seed/challenges/01-front-end-development-certification/html5-and-css.json [hunk 1]
```diff
@@ -32,9 +32,14 @@
...
-        "Cada desafio tiene pruebas que puedes ejecutar en cualquier momento presionado el botón \"Ejecutar pruebas\". Una vez logres pasar todas las pruebas, podrás avanzar al siguiente desafio.",
-        "Para pasar la prueba en este desafio, cambia tu texto de la etiqueta <code>h1</code> para que diga \"Hello World\" en lugar de \"Hello\". Entonces presiona el botón \"Ejecutar pruebas\"."
+        "La mayoría de los elementos HTML tienen una <code>etiqueta de apertura</code> y una <code>etiqueta de cierre</code>.", 
+        "Las etiquetas de apertura se ven como:", 
+        "<code>&#60;h1&#62;</code>",
+        " Las etiquetas de cierre se ven como:",
+        "<code>&#60;/h1&#62;</code>",
+        "Fíjate que la única diferencia entre las etiquetas de apertura y de cierre es que estas últimas tienen un / después de su signo de apertura (<).",
+        "Cada desafío tiene pruebas que puedes ejecutar en cualquier momento presionado el botón \"Ejecutar pruebas\". Una vez logres pasar todas las pruebas, podrás avanzar al siguiente desafío.",
```
Comment (vtamara) [inline seed/challenges/01-front-end-development-certification/html5-and-css.json]: Mejor lo que dice el boton "Run tests"

File: seed/challenges/01-front-end-development-certification/html5-and-css.json [hunk 2]
```diff
@@ -550,10 +549,8 @@
       "descriptionEs": [
         "Puedes establecer el tipo de letra de un elemento usando la propiedad <code>font-family</code>.",
         "Por ejemplo, si quieres establecer el tipo de letra de tu elemento <code>h2</code> como <code>Sans-serif</code>, usa el siguiente CSS:",
-        "<code>h2 {</code>",
-        "<code>&nbsp;&nbsp;font-family: Sans-serif;</code>",
-        "<code>}</code>",
-        "Haga que todos sus elementos <code>p</code> utilicen el tipo de letra <code>Monospace</code>."
+        "<blockquote>h2 {<br>&nbsp;&nbsp;font-family: Sans-serif;<br>}</blockquote>",
+        "Haga que todos tus elementos <code>p</code> utilicen el tipo de letra <code>Monospace</code>."
```
Comment (vtamara) [inline seed/challenges/01-front-end-development-certification/html5-and-css.json]: Haga->Haz

File: seed/challenges/01-front-end-development-certification/html5-and-css.json [hunk 3]
```diff
@@ -3347,8 +3311,8 @@
       "challengeType": 0,
       "nameEs": "Usa el código hexadecimal para colorear de blanco los elementos",
       "descriptionEs": [
-        "<code>0</code> es el dígito más bajo en código hexadecimal, y representa una completa ausencia de color.",
-        "<code>F</code> es el dígito más alto en código hexadecimal, y representa el máximo brillo posible.",
+        "<code>0</code> es el número más bajo en código hexadecimal, y representa una completa ausencia de color.",
```
Comment (vtamara) [inline seed/challenges/01-front-end-development-certification/html5-and-css.json]: Me parece mas preciso digito

**Comment by vtamara** [comment]
Few corrections. Good translation @justinian336 !

**Comment by justinian336** [comment]
I guess this one is done, @bugron.

**Comment by bugron** [comment]
@justinian336 @vtamara thank you, merged!

**Pull request merged**